Styles/style.css

```diff
@@ -0,0 +1,251 @@
+    margin: 0 auto;
+}
+
+.icon-section {
+    display: flex;
+    position: relative;
+}
+
+.txt-section {
+    display: flex;
+    flex-direction: column;
+}
+
+.navigation {
+    display: flex;
+    width: 90%;
+    height: 241px;
+    justify-content: center;
+    gap: 200px;
+    margin-top: 19px;
+}
+
+.nav-menu {
+    display: flex;
+    color: white;
+    gap: 100px;
+    margin-top: 22px;
+}
+
+.nav-menu a {
+    color: white;
+    text-decoration: none;
+}
+
+.about {
+    margin-top: 144px;
+    font-size: 100px;
+}
+
+/* TAGS */
+
+.main-tags {
+    display: flex;
+    margin-top: 74px;
+    gap: 10px;  
+    justify-content: center;
+}
+
+.tags {
+    color: #9A98AF;
+    border: 2.5px solid #F2F4F9;
+    box-sizing: border-box;
+    border-radius: 15px;
+}
+
+
+/* GAME BLOCK */
+
+.rating {
+    border-radius: 10px;
+    background-color: #363078;
+    padding: 10px 18px;
+    position: absolute;
+    margin: 10px;
+    color: white;
+}
+
+.game-block {
+    display: flex;
+    justify-content: center;
+    gap: 40px;
+    background-image: url(../Images/bg&polygons\ -\ games.png);
+    background-size: cover;
+    /* background-repeat: no-repeat; */
+}
+
+.game-block-left {
+    width: 560px;
+}
+
+.game-block-right {
+    width: 560px;
+}
+
+.game {
+    display: flex;
+    margin-top: 90px;
+    box-shadow: 0px 6px 20px rgba(61, 70, 118, 0.08);
+    border-radius: 30px;
+    background: #FFFFFF;
+    height: 380px;
+
+}
+
+.game-text {
+    text-align: center;
+    padding: 40px;
+}
+
+.game-img {
+    display: flex;
+    flex-direction: column;
+    margin-top: -30px;
+    margin-left: 30px;
+}
+
+.game-main-tags {
+    display: flex;
+    gap: 10px;
+    justify-content: center;
+    margin-top: 20px;
+}
+
+.game-tags {
+    background: #FFFFFF;
+    border: 2px solid #F2F4F9;
+    box-sizing: border-box;
+    border-radius: 10px;
+    padding: 16px;
+}
+
+
+/* FOOTER */
+
+.footer {
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+    background-image: url(../Images/Vector\ 17.png);
+    height: 502px;
+    width: 100%;
+    background-size: cover;
+}
+
+.footer-menu {
+    display: flex;
+    justify-content: center;
+    /* outline: 1px solid red; */
+    gap: 80px;
+}
+
+.footer-menu a {
+    color: white;
+    text-decoration: none;
+}
+
+
+/* PAGINATION */
+
+.pagination {
+    display: flex;
+    justify-content: center;
+}
+
+.pagination-ul {
+    display: flex;
+    align-items: center;
+    margin-bottom: 0;
+    padding-left: 0;
+    border-top: 1px solid #dee2e6;
+    padding-top: 15px;
+}
+
+.pagination .item {
+    display: inline-block;
+}
+
+.pagination .item .link {
+    padding: 6px 8px;
+    min-width: 32px;
+    min-height: 32px;
+    border-radius: 8px;
+    margin-left: 0;
+    margin-right: 8px;
+    border: 1px solid #fff;
+    font-size: 12px;
+    font-weight: 500;
+    color: #8A93A6;
+    display: inline-block;
+    text-align: center;
+    cursor: pointer;
```

Comment:
add one more property for centering numbers 
`line-height: 30px;`

Reply:
later

Reply:
changed to line-height: 24px;